src/assets/src/blocks/query-panel/index.js

```diff
@@ -0,0 +1,127 @@
+/**
```

Comment:
all these files within `/src/assets/src` are in the wrong path.  I also don't think we should bring these into these EE core branches from `eea-gutenberg` until we actually implement them.

This is also is a possible indicator that you've merged from a different branch or have NOT merged from the `FET/Gutenberg/11400/block-manager` branch that I took care of updating from master yesterday because in that branch I deleted all these files (or maybe you saw they were deleted and added them back?)

Reply:
branch just needed updating. is correct now